yb-voyager/src/queryparser/query_parser.go

```diff
@@ -0,0 +1,90 @@
+package queryparser
+
+import (
+	"fmt"
+
+	pg_query "github.com/pganalyze/pg_query_go/v5"
+	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+const (
```

Comment:
Define these in traversal.go? it is only used there.

Reply:
Also used in `unsupported_constructs.go`